src/content/en/updates/2017/03/chrome-58-media-updates.md

```diff
@@ -0,0 +1,221 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the Media updates in Chrome 58.
+
+{# wf_updated_on: 2017-03-21 #}
+{# wf_published_on: 2017-03-21 #}
+{# wf_tags: news,chrome58,media #}
+{# wf_featured_image: /web/updates/images/generic/animations.png #}
+{# wf_featured_snippet: Media controls customization, Autoplay for Progressive Web Apps added to Home screen, Pause autoplaying muted video when invisible, and color-gamut Media Query are there! #}
+
+# Media Updates in Chrome 58 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- Developers can now [customize media controls](#controlslist) such as the
+  download, fullscreen and remoteplayback buttons.
+- Sites installed using the "Add to Homescreen" flow can [autoplay audio
+  and video in the manifest's scope](#autoplay).
+- Chrome on Android now [pauses autoplaying muted video when invisible](#offscreen).
+- Developers can now access the approximate range of colors supported by Chrome and
+  output devices using the [`color-gamut` Media Query](#colorgamut).
+
+## Media controls customization {: #controlslist}
+
+Developers can now customize Chrome's native media controls such as the
+download, fullscreen and [remoteplayback] buttons using the new [ControlsList API].
+
+<div class="attempt-right">
+<figure>
+  <img src="/web/updates/images/2017/03/media-controls.png"
+       alt="Native media controls in Chrome 58">
+  <figcaption>
+    <b>Figure 1.</b>
+    Native media controls in Chrome 58
+  </figcaption>
+</figure>
+</div>
+
+This API aims to offer a way to control the native controls elements/buttons
```

Comment:
Maybe: 'This API offers a way to show or hide native media controls that do not make sense or are not part of the expected user experience, or only whitelist a limited set of features.'

• 'aims to offer a way' feels a bit uncertain (does it or doesn't it?)
• The 'some' in 'some native controls' sounds a bit vague — and I don't think it matters here if not all controls can be configured.
• 'limited amount of features' should be '... set ...' (amount makes more sense for mass/level, etc.) Also, it's about whitelisting certain features, not limiting the quantity of features.